docs/plugins/repository-s3.asciidoc

```diff
@@ -63,7 +59,7 @@ bin/elasticsearch-keystore add s3.client.default.secret_key
 ----
 
 The following are the available client settings. Those that must be stored in the keystore
-are marked as `Secure`.
+are marked as `Secure`; all other settings are public and may be included in the `elasticsearch.yml` file.
 
 `access_key`::
```

Comment:
In other docs (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/security-settings.html), we put the "secure" identifier directly after the setting name (instead of after the description) and it's a link to the secure settings page.  I suggest doing the same here.